apps/generic/ui/connectdialog.cpp

```diff
@@ -38,7 +38,7 @@ ConnectDialog::ConnectDialog(QWidget *parent) :
 {
 
     setAttribute(Qt::WA_DeleteOnClose, true);
-    setWindowTitle("KodiRemote - " + tr("Connect to XBMC"));
+    setWindowTitle("KodiRemote - " + tr("Connect to Kodi"));
```

Comment:
Shoud be Kodimote instead of KodiRemote. Most likely happened due to automatic rename by @mzanetti (changing Xbmc/XBMC to Kodi etc).